knowledge-base/scheduler-kb-context-menu-on-appointments-and-empty-slots.md

```diff
@@ -0,0 +1,367 @@
+                Start = DateTime.Today.AddDays(-10),
+                End = DateTime.Today.AddDays(-2),
+        },
+        new SchedulerAppointment
+        {
+                Title = "Jane's birthday party",
+                Description = "Make sure to get her fresh flowers in addition to the gift.",
+                Start = DateTime.Today.AddDays(5).AddHours(10),
+                End = DateTime.Today.AddDays(5).AddHours(18),
+        },
+        new SchedulerAppointment
+        {
+                Title = "Brunch with HR",
+                Description = "Performance evaluation of the new recruit.",
+                Start = DateTime.Today.AddDays(3).AddHours(3),
+                End = DateTime.Today.AddDays(3).AddHours(3).AddMinutes(45),
+        },
+        new SchedulerAppointment
+        {
+                Title = "Interview with new recruit",
+                Description = "See if John will be a suitable match for our team.",
+                Start = DateTime.Today.AddDays(3).AddHours(1).AddMinutes(30),
+                End = DateTime.Today.AddDays(3).AddHours(2).AddMinutes(30),
+        },
+        new SchedulerAppointment
+        {
+                Title = "New Project Kickoff",
+                Description = "Everyone assemble! We will also have clients on the call from a later time zone.",
+                Start = DateTime.Today.AddDays(3).AddHours(8).AddMinutes(30),
+                End = DateTime.Today.AddDays(3).AddHours(11).AddMinutes(30),
+        },
+        new SchedulerAppointment
+        {
+                Title = "Get photos",
+                Description = "Get the printed photos from last week's holiday. It's on the way from the vet to work.",
+                Start = DateTime.Today.AddHours(2).AddMinutes(15),
+                End = DateTime.Today.AddHours(2).AddMinutes(30),
+        },
+        new SchedulerAppointment
+        {
+                Title = "Conference",
+                IsImportant = true,
+                Description = "The big important work conference. Don't forget to practice your presentation.",
+                Start = DateTime.Today.AddDays(6),
+                End = DateTime.Today.AddDays(11),
+                IsAllDay = true,
+        }
+        };
+        return appointments;
+    }
+
+    public class ContextMenuItem
+    {
+        public string Text { get; set; }
+        public string CommandName { get; set; }
+        public ISvgIcon Icon { get; set; }
+        public bool Disabled { get; set; }
+    }
+
+    public class SchedulerAppointment
+    {
+        public Guid Id { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        public DateTime Start { get; set; }
+        public DateTime End { get; set; }
+        public bool IsAllDay { get; set; }
+        public bool IsImportant { get; set; }
+
+        public SchedulerAppointment()
+        {
+            this.Id = Guid.NewGuid();
+        }
+    }
+}
+
+````
```

Comment:
Add a `## See Also` section with relevant Context Menu and Scheduler links, even if they repeat ones from the steps.